docs/en/latest/terminology/global-rule.md

```diff
@@ -32,6 +32,12 @@ A [Plugin](./plugin.md) configuration can be bound directly to a [Route](./route
 
 Compared with the plugin configuration in Route, Service, Plugin Config, and Consumer, the plugin in the Global Rules is always executed first.
 
+::: note
+
+Global Rules do not apply for Stream Routes.
```

Comment:
Is this really true?
https://github.com/apache/apisix/blob/master/apisix/init.lua#L1042
https://github.com/apache/apisix/blob/master/apisix/init.lua#L461

Reply:
Oh, I missed this part of the code, however, stream routes would only cover the plugins in the `global_rules` that run on the `before_proxy` phase.

https://github.com/shreemaan-abhishek/apisix/blob/9276e64fbd967967c1c9d33e0bb443c21a6021cb/apisix/init.lua#L1058

Do you agree?

Reply:
Yeah